decidim-core/app/packs/src/decidim/sticky_header.js

```diff
@@ -0,0 +1,17 @@
+let prevScroll = window.scrollY;
+const mainBar = document.getElementById("main-bar");
+
+window.onscroll = function() {
```

Comment:
```suggestion
  document.addEventListener("scroll", () => {
```
Let's use the eventlistener here

Reply:
Done